src/github.ts

```diff
@@ -1333,6 +1291,21 @@ export class GitHub {
         pullRequest.labels
       );
 
+      // attempt to enable auto-merge
+      if (options?.autoMerge) {
+        try {
+          await this.enablePullRequestAutoMerge(
```

Comment:
If they don't have branch protection set up, should we simply attempt to directly merge it?

Reply:
If that's the user experience they expect, I would say yes. But that would mean any change result in a new release, which may or may not be what you want for customer SDKs.

Reply:
Ah, I see we do this in the method body, nevermind 👍